src/test/java/org/apache/commons/codec/digest/MurmurHash3Test.java

```diff
@@ -224,4 +225,238 @@ public void testSingleLongRandom() {
+
+      
+      private void doString(String s) {
+        doString(s, 0, 0);
+      }
+
+      private void doString(String s, int pre, int post) {
+        byte[] utf8 = s.getBytes(StandardCharsets.UTF_8);
+        int hash1 = MurmurHash3.hash32_x86(utf8, pre, utf8.length-pre-post, 123456789);
+        int hash2 = MurmurHash3.hash32_x86(s, pre, s.length()-pre-post, 123456789);
+        if (hash1 != hash2) {
+          System.out.println(s);
+          // second time for debugging...
+          hash2 = MurmurHash3.hash32_x86(s, pre, s.length()-pre-post, 123456789);
+        }
```

Comment:
The repeat of the test should be removed.

It is currently not executed due to a missing `@Test` annotation is the method that uses it.